.gitignore

```diff
@@ -5,4 +5,6 @@ staticfiles/
 docs/_build
 *.sqlite3
 .tox
-.coverage
\ No newline at end of file
+.coverage
+/.vscode/
+/node_modules/
```

Comment:
I think if you remove the slashes it will also work for packages/webapp/node_modules